recipes/mgclient/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class MGClientConan(ConanFile):
+    name = "mgclient"
+    description = "C/C++ Memgraph Client"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/memgraph/mgclient"
+    topics = ("memgraph", "client")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cpp": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cpp": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
```

Comment:
https://github.com/memgraph/mgclient/blob/master/CMakeLists.txt#L81
Seems like this would not be needed?

Not too sure how this works with the cpp bindings though

Reply:
@RubenRBS 
mgclient requires `optional` which is supported since C++17.
https://github.com/memgraph/mgclient/blob/master/mgclient_cpp/include/mgclient.hpp#L108